### PR TITLE
Add Helm release to staging

### PR DESCRIPTION
**Pull request by meyskens**
Signed-off-by: Maartje Eyskens <maartje@eyskens.me>

### CODE REVIEW

**Review by smacker — APPROVED** [review]
Does it require any new env vars in travis?

**Comment by meyskens** [comment]
@smacker it does, I have added them already as secrets

File: .travis.yml [hunk 1]
```diff
@@ -53,3 +53,7 @@ jobs:
       script:
         - PKG_OS=linux make packages
         - DOCKER_PUSH_LATEST=true make docker-push
+    - name: 'Deploy to staging'
+      stage: release
```
Comment (rporres) [inline .travis.yml:57]: I think it will be nicer to add an install phase that does a `make install-helm`

File: .travis.yml [hunk 2]
```diff
@@ -53,3 +53,7 @@ jobs:
...
+      script:
+        - HELM_RELEASE=lookout-gometalint-analyzer HELM_CHART=src-d/lookout-gometalint-analyzer K8S_NAMESPACE=lookout HELM_ARGS="--tiller-namespace=lookout --set image.tag=$TRAVIS_TAG -f .helm_staging.yml" make deploy
```
Comment (rporres) [inline .travis.yml]: I don't think this is going to work as we need the repo url. `HELM_CHART` should we set to  `lookout-gometalint-analyzer` and we should add `--repo https://src-d.github.io/charts/index.yaml`

**Review by rporres — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (meyskens) [reply .travis.yml]: 🤦‍♀️

**Review by rporres — APPROVED** [review]

**Pull request merged**